core/src/main/scala/kafka/server/ControllerApis.scala

```diff
@@ -99,6 +99,7 @@ class ControllerApis(val requestChannel: RequestChannel,
         case ApiKeys.INCREMENTAL_ALTER_CONFIGS => handleIncrementalAlterConfigs(request)
         case ApiKeys.ALTER_PARTITION_REASSIGNMENTS => handleAlterPartitionReassignments(request)
         case ApiKeys.LIST_PARTITION_REASSIGNMENTS => handleListPartitionReassignments(request)
+        case ApiKeys.ALTER_USER_SCRAM_CREDENTIALS => handleAlterUserScramCredentials(request)
```

Comment:
We cannot test it until we update ControllerApis to handle METADATA requests, so I'm going to wait on adding it.

Reply:
Yeah. Just to restate what I think we all know here, clients ask brokers to describe SCRAM credentials, rather than controllers. So this doesn't technically need to be implemented, except for testing.